R/pp_estimate.R

```diff
@@ -141,66 +119,66 @@ pp_estimate <- function(target, source, sid, spop, volume = NULL, ancillary = NU
     out <- pp_awi(target, source = source, sid = sid, spop = spop,
                   point = point)
   } else if (method == 'vwi') {
-    if (volume == 'NULL') {
-      usethis::ui_stop('volume is required for vwi')
+    if (is.null(volume)) {
```

Comment:
```suggestion
    if (volume == 'NULL') {
```

These are not equivalent